controllers/users.js

```diff
@@ -192,6 +192,32 @@ const getUsers = async (req, res) => {
       }
     }
 
+    const isDeparted = req.query.departed === "true";
+
+    if (isDeparted) {
+      let result;
+      if (dev) {
+        try {
+          result = await dataAccess.retrieveUsers({ query: req.query });
+          const departedUsers = await userService.getUsersWithIncompleteTasks(result.users);
+          if (departedUsers.length === 0) res.status(204).send();
```

Comment:
`
if (departedUsers.length === 0){
return res.status(204).send();
}
`
Can be written like this.